src/views/Maps.vue

```diff
@@ -8,7 +8,12 @@
     </p>
   </header>
 
-  <p>Voor niet-professionele gebruikers is er <a href="https://www.overstroomik.nl">www.overstroomik.nl</a> & <a href="https://www.klimaateffectatlas.nl/">www.klimaateffectatlas.nl</a>.</p>
+  <p>
+    Voor professionals die niet in de watersector werkzaam zijn en studenten is er
+    <a href="https://www.klimaateffectatlas.nl/" rel="noopener noreferrer" target="_blank">www.klimaateffectatlas.nl</a>,
```

Comment:
De links openen nu in een nieuw tabblad, is denk ik wel even het overleggen met Bart waard of dat gewenst is.